integration-tests/kafka-json-schema-apicurio2/pom.xml

```diff
@@ -13,6 +13,26 @@
     <name>Quarkus - Integration Tests - Kafka Json Schema with Apicurio 2.x</name>
     <description>The Apache Kafka Json Schema with Apicurio Registry 2.x integration tests module</description>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <artifactId>joda-time</artifactId>
+                <groupId>joda-time</groupId>
+                <version>2.10.14</version>
+            </dependency>
+            <dependency>
+                <groupId>org.jetbrains.kotlin</groupId>
```

Comment:
Why do we need this?

Reply:
Because the required dependency `kafka-json-schema-serializer` has two different versions of these three libraries, I must fix a compatible one. I can't do that at the project level, so I must fix the version at the module level in the integration tests.